modules/vectorfieldvisualizationgl/glsl/streamparticles.comp

```diff
@@ -89,16 +86,18 @@ void main() {
     uint gid = gl_GlobalInvocationID.x;
     vec3 p = pos[gid].xyz;
 
-    for (int i = 0; i < NUM_ADVECTIONS; i++) {
+    for (int i = 0; i < NUM_ADVECTIONS; i++) { 
         p = advect(p);
     }
 
-    float l = length(sampleVelocity(p));
-    if (l < 0.001) {
+    float v = length(sampleVelocity(p));
+    v = (v - minV) / (maxV - minV);
```

Comment:
might want to clamp [0 1] since the bounds are user specified and it might have side-effects for radius and color